tests/unit/EntityHeader.spec.ts

```diff
@@ -271,3 +271,55 @@ describe('Entity Header - AUTHORIZED TO CONTINUE OUT badge', () => {
     })
   })
 })
+
+describe('Entity Header - Operating Name', () => {
+  const router = mockRouter.mock()
+  it('displays operating name if firm', async () => {
+    // set store properties
+    businessStore.setBusinessInfo(
+      {
+        legalName: 'My Business',
+        alternateNames: [
+          {
+            operatingName: 'Wayne Enterprises'
```

Comment:
I'm confused why you don't need an identifier in here, but the test seems to be verifying what I would expect and it seems to pass, so 🤷‍♂️